lib/install/phlex.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+say "Installing Phlex..."
+
+application_configuration_path = Rails.root.join("config/application.rb")
+application_configuration_content = File.read(application_configuration_path)
+
+pattern = /config.autoload_paths << Rails.root.join\(["']app["']\)/
+
+unless application_configuration_content.match?(pattern)
+	inject_into_class(
+		application_configuration_path,
+		"Application",
+		%(    config.autoload_paths << Rails.root.join("app")\n)
```

Comment:
The Rails Guide [recommends](https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#config-autoload-paths) using interpolation here. I don't know if that's supported in older versions of Rails though.

```suggestion
		%(    config.autoload_paths << "#{root}/app"
```

Reply:
I just tested and `config.autoload_paths << "#{root}/app"` works in a Rails 6.1 application!

I just made the change to replace `config.autoload_paths << Rails.root.join("app")\n)` with `config.autoload_paths << "#{root}/app"`

I won't merge this PR immediately as I'll add the Tailwind config before. I'll re-request your review when I'm ready 😃